app/components/transaction/Result.tsx

```diff
@@ -33,17 +41,88 @@ export default function Result({ result }: any) {
   const execParams = result.transaction?.exec_transaction?.decoded?.parameters;
 
   // Mapping the parameters by name for easy access
-  const params = execParams ? execParams.reduce((acc: any, param: any) => {
-    acc[param.name] = param.value;
-    return acc;
-  }, {}) : {};
-
+  const params = useMemo(() => {
+    return execParams
+      ? execParams.reduce((acc: any, param: any) => {
+          acc[param.name] = param.value;
+          return acc;
+        }, {})
+      : {};
+  }, [execParams]);
+  
   const networkDetails = result.network?.name ?
     NETWORKS.find((n) => n.value === result.network.name || n.label === result.network.name) :
     null;
 
   const networkLogo = networkDetails?.logo || '';
 
+  useEffect(() => {
+    const newWarnings: {title: string, description: string}[] = [];
+    const newInfoNotices: {title: string, description: string}[] = [];
+
+    const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
I saw this value in different parts so is better to have a constant and export it.